app/createTournament/createTournamentDialog.js

```diff
@@ -0,0 +1,25 @@
+import * as tournamentValidation from './createTournamentBusiness.js';
+import {getListofPlayers} from '../playersinscription/addPlayerBusiness.js';
+import {Tournament} from '../Tournament.js'
+
+
+function showOrHideErrorMessage(validationResult) {
+	var $errorSummary = $('#errorsummary');
+
+	if (validationResult.success) {
+		$errorSummary.hide('slow');
+
+	} else {
+		var message = tournamentValidation.getErrorMessage(validationResult.error);
+		$errorSummary[0].innerText = message;
+		$errorSummary.show('slow');
+	}
+}
+
+export function createTournament(){
+ $(this).prop("disabled",true);
+ var validationResult=tournamentValidation.validateNumberofPlayers();
+ var listofPlayers=getListofPlayers();
```

Comment:
The list of players should not be something global, You should explicitly move it through modules. Don't use ES6 modules to end up working with global variables like the player list.